modules/router-store/migrations/migration.json

```diff
@@ -6,6 +6,11 @@
       "description": "The road to v6",
       "version": "5.2",
       "factory": "./6_0_0/index"
+    },
+    "ngrx-router-store-migration-02": {
+      "description": "The road to v8",
+      "version": "7.0",
```

Comment:
Should this be `8-beta` also?

Reply:
Yes, I updated the wrong one. `ngrx/store` is modified in #1716

Reply:
Modified in 29249e